stdlib/http/src/main/java/org/ballerinalang/net/http/HttpUtil.java

```diff
@@ -887,6 +889,23 @@ public static boolean isEntityDataSourceAvailable(BMap<String, BValue> struct) {
         return (entityStruct != null && EntityBodyHandler.getMessageDataSource(entityStruct) != null);
     }
 
+    /**
+     * Set the server name in the output response header.
+     *
+     * @param requestMsg  request message.
+     * @param outboundResponseMsg  outbound response message.
+     * @return outbound response message with a header server name.
+     */
+    private static void setHeaderToResponseMsg(HttpCarbonMessage requestMsg, HttpCarbonMessage outboundResponseMsg) {
+        if (requestMsg.getHeaders().contains(SERVER_NAME)) {
```

Comment:
Since we are setting header from `BallerinaHTTPConnectorListener`, this check is no longer needed right?